apps/files/src/components/FileList.vue

```diff
@@ -18,7 +18,7 @@
           <oc-table-group>
             <oc-table-row v-for="(item, index) in fileData" :key="index" :class="_rowClasses(item)" @click="selectRow(item, $event)" :id="'file-row-' + item.id">
               <oc-table-cell>
-                <oc-checkbox class="uk-margin-small-left" @click.stop @change.native="$emit('toggle', item)" :value="selectedFiles.indexOf(item) >= 0" />
+                <oc-checkbox class="uk-margin-small-left" @click.stop @change.native="$emit('toggle', item)" :value="selectedFiles.indexOf(item) >= 0" label="Select this item"/>
```

Comment:
Translate pls. Also, would it maybe make sense to write here something like "Select folder Photos" or something similar? The checkbox comes before the item name so the user doesn't know what he's selecting.

Reply:
Done (https://github.com/owncloud/phoenix/pull/2333/files#diff-4d43219c8fdf60539ad2d855346d33f2R196). I'm not sure if $gettext has to be refactored for this, because of different grammar rules/sentence structures in different languages. Ideally something like `this.$getttext('Select %s', item.name)` should work, but doesn't at the moment.

Reply:
See https://github.com/owncloud/phoenix/blob/9f50ba922e9e5430da460fa2c9ebc5337ab7e7a4/apps/files/src/mixins.js#L39 how to do this

Reply:
Done.